include/librealsense2/dds/dds-device-broadcaster.h

```diff
@@ -3,13 +3,14 @@
 
 #pragma once
 
+#include <librealsense2/dds/topics/device-info/device-info-msg.h>
```

Comment:
This includes FastDDS headers. We wanted to avoid including those in our header files.

Reply:
Hmm I can do it if I move `dds_device_handle` into the .cpp -- will do.

Reply:
Unfortunately, it does not work with gcc...
If you're using a broadcaster, you need to use the device-info.
I'm going to put it back the way it was, but I'll try to remove the need for the eProsima headers.